code/game/machinery/doors/door.dm

```diff
@@ -24,7 +24,7 @@
 	var/maxhealth = 300
 	var/health
 	var/destroy_hits = 10 //How many strong hits it takes to destroy the door
-	var/min_force = 10 //minimum amount of force needed to damage the door with a melee weapon
+	var/min_force = 20 //minimum amount of force needed to damage the door with a melee weapon

```

Comment:
This does, I meant to have this be for the blast doors. I'll have it changed in a bit.